sentinel-core/src/main/java/com/alibaba/csp/sentinel/annotation/SentinelResource.java

```diff
@@ -63,4 +59,9 @@
      * @return name of the fallback function, empty by default
      */
     String fallback() default "";
+
+    /**
+     * @return the exception classes to trace, Throwable.class by default
+     */
+    Class<? extends Throwable>[] exceptionClasses() default {Throwable.class};
```

Comment:
`exceptionClasses` might not be clear enough. Maybe it's better to be `exceptionsToTrace`?

Reply:
Agree with your proposal.